__mocks__/react-native-onyx.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable rulesdir/prefer-onyx-connect-in-libs */
```

Comment:
Can we get rid of this, or add a comment explaining why we need it here?

Reply:
We can't get rid of this but I've added a comment explaining the reason we are disabling the rule.
Please have a look.